core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.html

```diff
@@ -45,17 +45,17 @@ <h4 class="col oppia-translation-modal-author-header">Submitted by {{ authorName
             </li>
           </ul>
           <schema-based-editor [schema]="getHtmlSchema.bind(this)"
-                               *ngIf="startedEditing && isHtmlContent"
+                               *ngIf="startedEditing && htmlIsContent"
```

Comment:
This changes the meaning (which is perhaps why the original name was ambiguous).

It should be contentTypeIsHtml. Similarly below for others.

Reply:
Done